client/tailscale/tailscale_test.go

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+package tailscale
+
+import "testing"
+
+func TestClientBuildURL(t *testing.T) {
+	c := Client{BaseURL: "http://127.0.0.1:1234"}
+	got := c.BuildURL("devices")
+	want := "http://127.0.0.1:1234/api/v2/devices"
```

Comment:
const want = …

Should we also have a test that, e.g., spaces and other characters get properly escaped?